macros/contexts/contextReaction.pl

```diff
@@ -112,6 +113,48 @@ =head1 DESCRIPTION
 
 to allow a state of C<(x)> for a compound.
 
+By default, the Reaction context checks the student answer against the
+correct answer in the form the correct answer is given (molecular or
+condensed structural form).  If you want to allow either form, then
+you can set the flag C<acceptMolecularForm> to C<1>, in which case,
+the student's answwer is compared first to the original correct answer
+and then to its molecular form.  For example, if the correct answer is
+`(NH_4)_3PO_4`, then a student answer of `N_3H_12PO_4` would also be
```

Comment:
It looks like you fell into markdown mode here!  This should probably be
```suggestion
C<(NH_4)_3PO_4>, then a student answer of C<N_3H_12PO_4> would also be
```